### ISSUE TITLE
Allow ability to build/install only the CLI interface

### ISSUE THREAD

**Issue by cslycord** [riverrun/genxword#22]
It would be greate to add the ability to only build the CLI interface/program. Personally, I mostly use this program/package at work through Cygwin/WSL on windows without any GUI and have no need for the GTK side of things.

I'd like the ability to build the program to not have that part and to not need the GTK dependencies. An obvious workaround is just have it build with the deps then delete the files I don't need, but I'd prefer the ability to have it trimmed down when I build it to begin with.

Thanks

**Comment by riverrun** [comment]
Thanks for raising the issue. I will look into it at the weekend and try to work out how feasible it is.

**Comment by kmccurley** [comment]
This is _not_ the year of the linux desktop, and I cannot install this on ubuntu 18.04 because of mysterious unresolved dependencies. Removal of the GUI would be highly desirable, because most of the value is in the code to generate the layout, not the UI.

**Comment by riverrun** [comment]
Yes, I would like to create a 'lite' version, which does not need a lot of the dependencies. The problem at the moment is just getting the time to do so.

I will add a note to this issue if and when I get the chance to work on this.
